contrib/ansible/roles/node/tasks/generic-install.yml

```diff
@@ -1,3 +1,10 @@
+- name: Install Docker
```

Comment:
There is a role for installing and configuring Docker. It should be ran when installing the `node` role as a dependency. Is this not working for you?